pkg/apiserver/info/info.go

```diff
@@ -45,8 +45,9 @@ func Register(r *gin.RouterGroup, auth *user.AuthService, s *Service) {
 }
 
 type InfoResponse struct { //nolint:golint
-	Version    pkgutils.VersionInfo `json:"version"`
-	PDEndPoint string               `json:"pd_end_point"`
+	Version      pkgutils.VersionInfo `json:"version"`
+	PDEndPoint   string               `json:"pd_end_point"`
+	EnableReport bool                 `json:"enable_report"`
```

Comment:
As a default opt-in option, prefer to call it `DisableXxx`, like `DisableTelemetry`.